clouddq/main.py

```diff
@@ -199,10 +199,11 @@
 )
 @click.option(
     "--enable_experimental_dataplex_gcs_validation",
-    help="If True, allows validating Dataplex GCS resources using "
-    "BigQuery External Tables",
+    help="This flag is set to True by default, it allows validating Dataplex GCS resources using "
```

Comment:
nit: "This flag has no effect and is now deprecated. GCS resource validation via BigQuery External Tables is supported regardless of the value of this flag. Please remove it from your script."

Reply:
Is it only for GCS resource attached to Dataplex, or any GCS resource behind BigQuery external table?